README.md

```diff
@@ -245,13 +245,32 @@ The figure shows the flow of execution in the cluster mode. When users run code
 
 ### Creating a session
 
-To use GraphScope in a distributed setting, we need to establish a session in a python interpreter. 
+To use GraphScope in a distributed setting, we need to establish a session in a python interpreter.
+
+For convenience, we provide several demo datasets, and an option `with_demo_dataset` to mount the dataset in the graphscope cluster. The datasets will located in `/dataset`.

```

Comment:
... The datasets will mount to `~/dataset` in the pods as default, or you can assign any other location. If you want to use your own data on k8s cluster, please refer to [this](#).